ThreeBotPackages/zerobot/chat/templates/chat/login.html

```diff
@@ -1,7 +1,7 @@
 {% extends "chat/home.html"%}
 {% block title %}Login{% endblock %}
 {% block content %}
-<h2>Please select one of the following providers to login.</h2>
+<h2>Please login by press on 3bot image .</h2>
```

Comment:
`press` should be `pressing`. 
However, maybe the image/button should have a Login label so you don't have to explicityl ask a user to press on anything.